src/drivers/nuttx/lv_nuttx_fbdev.c

```diff
@@ -138,21 +139,33 @@ int lv_nuttx_fbdev_set_file(lv_display_t * disp, const char * file)
     uint32_t w = dsc->vinfo.xres;
     uint32_t h = dsc->vinfo.yres;
     uint32_t stride = dsc->pinfo.stride;
-    uint32_t data_size = h * stride;
+    size_t data_size = h * stride;
```

Comment:
why not size_t, uint32_t is ok
```suggestion
    uint32_t data_size = h * stride;
```

Reply:
Done